avoid_planner:
  # 传感器参数
  max_sensor_range: 50.0
  min_sensor_range: 0.5
  voxel_grid_size: 0.1
  statistical_filter_mean_k: 10
  statistical_filter_std_dev: 0.1
  lidar_frame_id: "livox_frame"
  body_frame_id: "base_link"
  lidar_topic: "/livxxox/lidar"

  # 直方图参数
  azimuth_resolution_deg: 1.0  # 方位角步长（度）
  elevation_resolution_deg: 5.0  # 俯仰角步长（度）
  min_azimuth_deg: -180.0  # 最小方位角
  max_azimuth_deg: 180.0  # 最大方位角
  min_elevation_deg: 0.0  # 最小俯仰角
  max_elevation_deg: 30.0  # 最大俯仰角
